debian/sid/debian/control

```diff
@@ -51,6 +51,10 @@ Build-Depends: cmake,
      libsimbody-dev (>= 3.0.0),
      libsimbody-dev (<< 4.0.0),
      libbullet-dev,
+# dart without major version number to get versions coming from Debian/Ubuntu upstream
```

Comment:
There's no DART upstream for Bionic, right? In that case, what version will this be linked against, the one from DART's PPA or ours?

Reply:
When we release we never relay on third party repositories or PPAs, just Ubuntu/Debian official packages + our packages.osrfoundation.org repositories (occasionally packages.ros.org repos).

Reply:
Ok. Sorry, I think I've asked this before, but what will happen if someone installs `gazebo11` on Bionic and they also have DART's PPA? Will the version from the PPA be uninstalled?

Reply:
The one that is greater while in line with the restriction < 6.10.0